packages/SwingSet/demo/encouragementBot/bootstrap.js

```diff
@@ -1,4 +1,3 @@
-/* global harden */
 import { E } from '@agoric/eventual-send';
```

Comment:
Hm, pity, my main editor (`js2-mode` on emacs) knows about the `/* global` comment but not about external `.eslintrc` files. I'll look for a different way to teach it about globals.

I assume eslint is now complaining about redundant `/* global */` declarations?

Reply:
> I assume eslint is now complaining about redundant /* global */ declarations?

Yep.